chains/ContractsRococo.ts

```diff
@@ -0,0 +1,15 @@
+import { IChain } from './types.ts';
+
+export const ContractsRococo: IChain = {
```

Comment:
I don't have exp with Rococo, just with Aleph and Astar, but the `Contracts` prefix looks odd, there might be reasons for this, but on the first glance it looks weird. I guess they have this endpoint as well `wss://rococo-rpc.polkadot.io`

Reply:
Rococo and Contracts on Rococo are two different chains :). Rococo does not have `pallet-contracts`, but Contracts on Rococo does. I should add Rococo as well to avoid confusion. 

Thanks for pointing that out

Reply:
Thx for the explanation, I knew that there was a reason behind that naming.